ch_trees/gui.py

```diff
@@ -170,27 +167,49 @@ class TreeGen(bpy.types.Operator):
     _scene.tree_blossom_rate_input = _props.FloatProperty(name="", description="Fractional rate at which blossom occurs relative to leaves", default=0, min=0, max=1)
 
     # Save location for custom tree params
-    _scene.custom_tree_save_overwrite_input = _props.BoolProperty(name="Overwrite if File Exists", default=False)
+    _scene.custom_tree_save_overwrite_input = _props.BoolProperty(name="Overwrite If File Exists", default=False)
     tree_save_location = os.path.sep.join((_get_addon_path_details()[2], 'parametric', 'tree_params', 'my_custom_tree.py'))
     _scene.custom_tree_save_location_input = _props.StringProperty(name="", default=tree_save_location)
 
+    # ----
+    # Utilities
+
     # Load custom params
     _scene.custom_tree_load_params_input = _props.EnumProperty(name="", default="ch_trees.parametric.tree_params.quaking_aspen", items=parametric_items)
 
+    # Render inputs; auto-fill path input with user's home directory
+    _scene.render_input = _props.BoolProperty(name="Render", default=False)
```

Comment:
I feel like this should be "Render after generation" or something to make clear it's automatic.